GVRf/Framework/framework/src/main/java/org/gearvrf/GVREyePointeeHolder.java

```diff
@@ -184,11 +101,11 @@ public void setEnable(boolean enable) {
      *            The {@link GVREyePointee} to add
      * 
      */
-    public void addPointee(GVREyePointee eyePointee) {
+    public void addCollider(GVRCollider eyePointee) {
```

Comment:
Since this whole class is deprecated does it deserve having the new add/removeCollider apis?